source/includes/api/_subscriptions.md

```diff
@@ -1,704 +1,811 @@
-application   | Application of the subscription. Set to `gfw` by default          |             String | No
-language      | Language of the subscriptions (used to select the email template) | en, es, fr, pt, zh | Yes
-resource      | Details on the resource that will be notified for the subscription. |           Object | Yes
--- type       | The type of resource to notify. If `EMAIL`, an email is sent to the email saved in the resource content. If `URL`, a POST is requested to the web-hook URL in the resource content. | String | Yes
--- content    |  The email or URL that will be notified (according to the type).  |             String | Yes
-datasets      |               Array of datasets of the subscription               |              Array | Yes (unless `datasetsQuery` is specified)
-datasetsQuery |              Subscriptions to subscribable datasets               |              Array | Yes (unless `datasets` is specified)
--- id         |                           Id of dataset                           |           ObjectId | Yes (unless `datasets` is specified)
--- type       | Type of subscription defined in the dataset                       |               Text | Yes (unless `datasets` is specified)
--- params     | Geographic area of the subscription                               |             Object | Yes (unless `datasets` is specified)
-env           |  Environment of the subscription. Set to `production` by default  |             String | No
+```shell
+curl -X GET "https://api.resourcewatch.org/v1/subscriptions/<subscription_id>/confirm"
+```
 
-## Unsubscribe
+Upon creation of the subscription, a confirmation email is sent to the email of the user who created the subscription. This email will contain a link, which you will need to click in order to confirm the subscription. You can also confirm the subscription by performing a GET request to the `v1/subscriptions/:id/confirm` endpoint, as exemplified on the side.
 
-To unsubscribe from a subscription:
+*Manually calling this endpoint will redirect you to the GFW application.*
 
-```shell
-curl -X GET https://api.resourcewatch.org/v1/subscriptions/:id/unsubscribe \
--H "Authorization: Bearer <your-token>"
-```
+### Resending the subscription confirmation email
 
-## Delete subscription
+> Example GET request to resend the confirmation email for a subscription:
 
 ```shell
-curl -X DELETE https://api.resourcewatch.org/v1/subscriptions/:id/unsubscribe \
+curl -X GET "https://api.resourcewatch.org/v1/subscriptions/<subscription_id>/send_confirmation" \
 -H "Authorization: Bearer <your-token>"
 ```
 
-To delete a subscription, use the following DELETE endpoint.
+For convenience, the RW API offers an additional endpoint to resend the confirmation email. In order to do this, you should perform a GET request to the `v1/subscriptions/:id/send_confirmation`. As with most of the other subscription endpoints, please keep in mind that you must be authenticated in order to use this endpoint.
```

Comment:
What happens if the subscription is already confirmed and I accidentally request another confirmation email? Will it still come? Will the confirmation get switched back to false until I click the link?

Reply:
No changes are performed to the subscription when triggering another confirmation email (i.e. if the subscription was already confirmed, it stays that way). 

The email will always be sent, regardless of the status of the subscription (confirmed or not).

I will add this info to the endpoint docs, thanks for pointing this out :)